### PR TITLE
Improved structure

### PR DESCRIPTION
**Pull request by tomasvanrijsse**
Today in the presentations of the project I heard some students about the file structure of the project.
It took some of them one or even two days to get started because the new structure was daunting.

1. I've added a video to explain the structure
2. I've removed the router, the project is easier to understand without it

Since all projects look the same it became clear to me that students took the example really serious.
The list of tasks was taken as the actual requirements of the project.
And the project that was shown by Rob at the first Sunday was mimicked as good as the students could.
Therefore I changed the last part of the readme and added a few simple examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,44 +31,43 @@ src
...
-- [ ] A user can see their score at the end of the quiz
+Choose a desired end result at the start of the project.
+By choosing an existing example, the goal of the project will be clear to everyone in the team.
+Another benefit is that it will be easier to come up with the necessary tasks to create your quiz.
 
-## Nice-to-Haves
+Some examples of online quizes that you might like:
 
-> if time, add more features!
+- https://quiza.nl/raad-de-chips
```
Comment (robvk) [inline README.md:55]: This is a single page quiz so may get misinterpreted. Did a quick search, what do you think about:

- https://www.wizardingworld.com/quiz/the-first-year-hogwarts-houses-quiz
- https://openpsychometrics.org/tests/HEMCR/1.php
- https://www.lenstore.co.uk/vc/colour-is-in-the-eye-of-the-beholder/#/game
- https://heywise.com/quiz/lets-find-out-if-you-follow-gordon-ramsay/1/

File: README.md [hunk 2]
```diff
@@ -16,6 +16,11 @@ There are no dependencies needed to run the website, everything is prepared to w
 
 # Structure
 
+Instead of writing all code in a single JavaScript file, we want you to split your code over several files.
+The structure of this project is explained in the next video
+
+[![Project Structure YouTube Video](https://i.imgur.com/hDcLYFt.png)](https://youtu.be/bysBqtSKBpQ)
```
Comment (robvk) [inline README.md:22]: Really cool video, great idea starting with a single file and doing the splitting live! Thanks a lot for this

**Review by robvk — CHANGES_REQUESTED** [review]
The video is amazing!

I can understand wanting the teams to copy an end result. Usually I like giving them freedom so that they can go as far as they can, but giving them a goal will probably make it easier on them.

All great, only want to change the examples a little bit. Thoughts on the alternatives?

File: README.md (hunk 1 above)
Comment (tomasvanrijsse) [reply README.md:55]: I like your examples as well.
I'd only remove the psychological test, its UI isn't examplary and the theme a bit complicated.

**Comment by tomasvanrijsse** [comment]
I think although we give them freedom, they seek for well defined requirements. Most teams therefore, stick to the format shown during the first meeting. Giving more examples should explain them that they actually have some freedom, i hope 🤷‍♂️

**Comment by robvk** [comment]
Yeah, you may be right. Let's try it and see what happens! Will merge and then add the other examples :+1:

**Pull request merged**